IdentityCore/src/broker_operation/request/browser_native_message_request/MSIDBrowserNativeMessageGetTokenRequest.h

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.  
+
+
+#import "MSIDBrowserNativeMessageRequest.h"
+
+@class MSIDAADAuthority;
+@class MSIDAccountIdentifier;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSIDBrowserNativeMessageGetTokenRequest : MSIDBrowserNativeMessageRequest
+
+/// uid.utid
+@property (nonatomic, nullable) MSIDAccountIdentifier *accountId;
+
+/// Identifies an application that requests a token.
+@property (nonatomic) NSString *clientId;
+
+///If it is passed, broker will respect authority, otherwise broker will use default authority.
+@property (nonatomic, nullable) MSIDAADAuthority *authority;
+
+/// List of scopes.
+@property (nonatomic) NSString *scopes;
+
+/// The redirect uri.
+@property (nonatomic) NSString *redirectUri;
+
+/// Indicates the type of user interaction that is required. Valid values are login, none, consent, and select_account.
+@property (nonatomic, nullable) NSString *prompt;
```

Comment:
should this be nonnull?

Reply:
According to the current Browser SSO protocol it is optional.